tests/InputFiles/GeneralizedHarmonic/BinaryBlackHole.yaml

```diff
@@ -29,9 +29,11 @@ DomainCreator:
       UseLogarithmicMap: false
     EnvelopingCube:
       Radius: 100.0
-    OuterSphere:
-      Radius: 1493.0
-      UseLogarithmicMap: false
+      UseProjectiveMap: true
+    OuterShell:
+      InnerRadius: Auto
+      OuterRadius: 1493.0
```

Comment:
Can this be rounded to 1500? Or does this/these numbers come from SpEC?

Reply:
It's what Geoffrey put here, so I don't want to just change it :) Probably from SpEC